modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/domain/delegatedStake/UpdateDelegatedStakeAcceptanceResult.scala

```diff
@@ -0,0 +1,18 @@
+package io.constellationnetwork.node.shared.domain.delegatedStake
+
+import cats.data.NonEmptyChain
+
+import scala.collection.immutable.SortedMap
+
+import io.constellationnetwork.node.shared.domain.delegatedStake.UpdateDelegatedStakeValidator.UpdateDelegatedStakeValidationError
+import io.constellationnetwork.schema.SnapshotOrdinal
+import io.constellationnetwork.schema.address.Address
+import io.constellationnetwork.schema.delegatedStake.UpdateDelegatedStake
+import io.constellationnetwork.security.signature.Signed
+
+case class UpdateDelegatedStakeAcceptanceResult(
+  acceptedCreates: SortedMap[Address, List[(Signed[UpdateDelegatedStake.Create], SnapshotOrdinal)]],
+  notAcceptedCreates: List[(Signed[UpdateDelegatedStake.Create], NonEmptyChain[UpdateDelegatedStakeValidationError])],
```

Comment:
as above, it should not be a tuple but a list of validated elements (all invalid in that case). so you can take the value from it or validation errors.